### ISSUE TITLE
pip install failure on 0.5 release

### ISSUE THREAD

**Issue by hanc1208** [gruns/furl#75]
```
  Using cached furl-0.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/12/y2p8nk8s4hx_2x79q6rws8780000gn/T/pip-build-7kf0kjut/furl/setup.py", line 26
        print 'python setup.py sdist'
                                    ^
    SyntaxError: Missing parentheses in call to 'print'
    
    ----------------------------------------
```
Cannot install furl>=0.5 using pip

**Comment by gruns** [comment]
Fixed in v0.5.1. Upgrade with

```
pip install furl --upgrae
```

Thank you for bringing this issue to my attention so quickly Han. Don't hesitate to let me know if there's anything else I can do for you.

**Comment by hanc1208** [comment]
Thank you for your great library.
